### PR TITLE
Implement Hyperlink to the currents IconPack Website

### LINKED ISSUE #227: Add Hyperlinks to the IconPack's website
**Is your feature request related to a problem? Please describe.**
It would be cool if there is a link to each website of the IconPack. 

**Describe the solution you'd like**
Add the links and provide a Hyperlink-Button in the Browser

**Describe alternatives you've considered**
Using a search engine

**Additional context**
I'll make a proposal for a few iconPacks as a draft PR and update the other ones if you consider my idea as useful.

### PR DESCRIPTION
**Pull request by timunie**
## What has changed?
You can now link the IconPack to its original website. This should make the IconPacks more known to new users and is IMO a benefit for the IconPack-Maintainer. 

## What is the current state?
I have implement it for the first three `IconPacks`. If I get an 👍 	I'll update the others as well. 

## Preview
image

## Closed issues
Closes #227

### CODE REVIEW

**Comment by timunie** [comment]
## Update
With #231 we have now a very generic solution. This solution is now implemented into the Browser:
- Use the new `MetaDataAttribute` to get the URLs
- Added a Button to show the License as well as the Projects Website

## Screenshot
image

File: src/MahApps.Metro.IconPacks.Browser/Styles/CustomTabControl.xaml [hunk 1]
```diff
@@ -226,34 +401,61 @@
...
+                               Text="{Binding Caption, StringFormat='{}{0} icon collection'}" />
+                    <Button Margin="5,5,0,5"
+                            VerticalAlignment="Top"
+                            Command="{x:Static viewModels:MainViewModel.OpenUrl_Command}"
+                            CommandParameter="{Binding ProjectUrl}"
+                            Content="{iconPacks:FontAwesome Kind=ExternalLinkAltSolid}"
+                            Foreground="{DynamicResource MahApps.Brushes.Gray2}"
+                            Style="{DynamicResource MahApps.Styles.Button.WindowCommands}">
+                        <Button.ToolTip>
+                            <TextBlock Text="{Binding ProjectUrl, StringFormat='{}Visist: {0}'}" />
```
Comment (jizc) [inline src/MahApps.Metro.IconPacks.Browser/Styles/CustomTabControl.xaml:425]:
```suggestion
                            <TextBlock Text="{Binding ProjectUrl, StringFormat='{}Visit: {0}'}" />
```

Comment (timunie) [reply src/MahApps.Metro.IconPacks.Browser/Styles/CustomTabControl.xaml]: @jizc thank you for fixing that. :+1:

**Comment by punker76** [comment]
@timunie your changes have been merged, thanks for your contribution 👍

**Pull request merged**